docs/source/distributed_simulation.rst

```diff
@@ -50,19 +51,19 @@ RPCs and Replication
 
 Communication between Server and Client uses UE’s `replication and RPCs <https://docs.unrealengine.com/4.27/en-US/InteractiveExperiences/Networking/Actors/>`_.
 
-**Replication** is the process to sync data between the server and the clients. Changes of Actors and properties which set as "replciated" in server are synced to the cliets.
-Replciation is the process from server to process to sync data such as Actor pose, and other properties.
+**Replication** is the process to sync data between the server and the clients. Changes of Actors and properties set as "replicated" in server are synced to the clients.
+Replication is the process of the server to sync data such as Actor pose, and other properties.
```

Comment:
I want to emphasize Replication is a one-way process from server to client

Reply:
Yes, then I would propose for the last sentence: "Replication is a one-way process from server to client to sync data such as Actor pose and other properties."